spec/api/v1/external_clearstream_list_request_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require '/home/lex/Clients/Concur/Projects/stub_api/spec/helpers.rb'
+
+RSpec.configure do |c|
+  c.include Helpers
+end
+
+RSpec.describe 'External list requests to Clearstream', type: :request do
+  it 'get all lists' do
```

Comment:
```suggestion
  it 'gets all lists' do
```

Reply:
Let's clean up the tests in the next feature branch, okay?